test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -235,7 +278,14 @@
    #(es-index/delete! % "ctia_*")
    (helpers/fixture-ctia-with-app
     (fn [app]
-      (helpers/set-capabilities! app "foouser" ["foogroup"] "user" (capabilities/all-capabilities))
-      (whoami-helpers/set-whoami-response app "45c1f5e3f05d0" "foouser" "foogroup" "user")
+      (helpers/set-capabilities!
+       app "foouser" ["foogroup"] "user" (capabilities/all-capabilities))
+      (whoami-helpers/set-whoami-response
+       app "45c1f5e3f05d0" "foouser" "foogroup" "user")
       (doseq [test-case (test-cases)]
-        (test-search-case app test-case))))))
+        (test-search-case
+         app
+         test-case
+         ;; `version` is an anaphoric var interned by `for-each-version` macro
+         ;; please disregard concerning linter errors
+         :es-version version))))))
```

Comment:
If you prefer to limit the pb with the linter here, you can retrieve this value from the config.